Mathlib/Topology/Algebra/Module/ModuleTopology.lean

```diff
@@ -134,6 +134,14 @@ theorem eq_moduleTopology [τA : TopologicalSpace A] [IsModuleTopology R A] :
     τA = moduleTopology R A :=
   IsModuleTopology.eq_moduleTopology' (R := R) (A := A)
 
+/--
+Note that the topology isn't part of the discrimination key so this gets tried on every
+`IsModuleTopology` goal and hence the low priority.
+-/
+instance (priority := low) {R : Type*} [TopologicalSpace R] {A : Type*} [Add A] [SMul R A] :
+    letI := moduleTopology R A; IsModuleTopology R A :=
+  letI := moduleTopology R A; ⟨rfl⟩
+
```

Comment:
Another possibility here is that you just make this a local instance in the `Adjunction` section of `Algebra/Category/ModuleCat/Topology/Basic.lean`; this is enough to make the file compile. Do you think your approach is better or might it lead to trouble? In my work on the module topology I've just been doing
```lean
    letI := moduleTopology R X
    haveI : IsModuleTopology R X := ⟨rfl⟩
```
when I run into this, but it seems harder to do this in the application in this PR because X is some unwieldy thing.

Other than this comment this PR looks fine now. What do you think?

Reply:
I don't think this would lead to much trouble and having this instance is fine.